src/DssSpell.sol

```diff
@@ -48,134 +57,180 @@ contract DssSpellAction is DssAction {
+        //       However the active bug bounty programs still need to be updated to reflect that,
+        //       so we are authorizing the ESM on the relevant components.
 
-        // Cloaky - 20.40 MKR - 0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818
-        MKR.transfer(CLOAKY, 20.40 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+        // Auth ESM on MCD_LITE_PSM_USDC_A_IN_CDT_JAR
+        DssExecLib.authorize(MCD_LITE_PSM_USDC_A_IN_CDT_JAR, MCD_ESM);
 
-        // PBG - 16.58 MKR - 0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2
-        MKR.transfer(PBG, 16.58 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+        // ---------- Add LitePSM keeper network job ----------
+        // Forum: https://forum.makerdao.com/t/lite-psm-usdc-a-phase-1-test-period-proposed-parameters/24644
+        // Poll: https://vote.makerdao.com/polling/QmdcHXHy
 
-        // BONAPUBLICA - 13.89 MKR - 0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3
-        MKR.transfer(BONAPUBLICA, 13.89 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+        // sequencer.addJob( 0x689cE517a4DfCf0C5eC466F2757D324fc292C8Be )
+        // Note: the parameters below are set in `CRON_LITE_PSM_JOB` constructor
+        // fill: Set threshold at 15M DAI
+        // trim: Set threshold at 30M DAI
+        // chug: Set threshold at 300k DAI
+        DssCronSequencerLike(CRON_SEQUENCER).addJob(CRON_LITE_PSM_JOB);
 
-        // Rocky - 13.89 MKR - 0xC31637BDA32a0811E39456A59022D2C386cb2C85
-        MKR.transfer(ROCKY, 13.89 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+        // ---------- Chainlog additions ----------
+        // Forum: https://forum.makerdao.com/t/lite-psm-usdc-a-phase-1-test-period-proposed-parameters/24644
 
-        // vigilant - 12.55 MKR - 0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61
-        MKR.transfer(VIGILANT, 12.55 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+        // Add 0x69cA348Bd928A158ADe7aa193C133f315803b06e as MCD_LITE_PSM_USDC_A_JAR
+        DssExecLib.setChangelogAddress("MCD_LITE_PSM_USDC_A_JAR", MCD_LITE_PSM_USDC_A_JAR);
 
-        // Ennoia (Cloaky) - 4.10 MKR - 0xA7364a1738D0bB7D1911318Ca3FB3779A8A58D7b
-        MKR.transfer(CLOAKY_ENNOIA, 4.10 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+        // Add 0x5eeB3D8D60B06a44f6124a84EeE7ec0bB747BE6d as MCD_LITE_PSM_USDC_A_IN_CDT_JAR
+        DssExecLib.setChangelogAddress("MCD_LITE_PSM_USDC_A_IN_CDT_JAR", MCD_LITE_PSM_USDC_A_IN_CDT_JAR);
 
-        // Kohla (Cloaky) - 4.10 MKR - 0xA9D43465B43ab95050140668c87A2106C73CA811
-        MKR.transfer(CLOAKY_KOHLA, 4.10 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+        // Add 0x689cE517a4DfCf0C5eC466F2757D324fc292C8Be as CRON_LITE_PSM_JOB
+        DssExecLib.setChangelogAddress("CRON_LITE_PSM_JOB", CRON_LITE_PSM_JOB);
```

Comment:
Since every new LitePSM would require a different cron job (with different tresholds, but more importantly, [different LitePSM address](https://github.com/makerdao/dss-cron/blob/2f9aba4ab16603a77b55333b8ebaf348f5fe7e47/src/LitePsmJob.sol#L36)) I believe a more suitable name would be `CRON_LITE_PSM_USDC_A_JOB`.

Reply:
We'll probably have to replace that job with one that can take care of multiple PSMs, otherwise we'd be using 1 slot for every deployed PSM, which is not desirable.

Reply:
This implies we will have a "configurable job" with auth as we would most certainly need different thresholds per PSM. Not sure why it wasn't developed like so since the beginning then.

In any case, I think it's fine to keep it as it's just a non-blocking naming issue and might otherwise delay the spell. We can rename it as soon as the new LitePSM module is onboarded (and the updated job structure is settled).

Reply:
> Not sure why it wasn't developed like so since the beginning then.
Initially there weren't supposed to be multiple instances of LitePSM, however that requirement changed recently.
The job code was written to comply with a single LitePSM and there has been no time to update it since the change.

> This implies we will have a "configurable job" with auth as we would most certainly need different thresholds per PSM. 
Yes, before we onboard a new LitePSM, the job will need to be reworked.